src/Microsoft.TestPlatform.Build/Tasks/VSTestTask.cs

```diff
@@ -161,6 +163,27 @@ public override bool Execute()
             var traceEnabledValue = Environment.GetEnvironmentVariable("VSTEST_BUILD_TRACE");
             Tracing.traceEnabled = !string.IsNullOrEmpty(traceEnabledValue) && traceEnabledValue.Equals("1", StringComparison.OrdinalIgnoreCase);
 
+            var debugEnabled = Environment.GetEnvironmentVariable("VSTEST_BUILD_DEBUG");
+            if (!string.IsNullOrEmpty(debugEnabled) && debugEnabled.Equals("1", StringComparison.Ordinal))
+            {
+                Console.WriteLine("Waiting for debugger attach...");
+
+                var currentProcess = Process.GetCurrentProcess();
+                Console.WriteLine(string.Format("Process Id: {0}, Name: {1}", currentProcess.Id, currentProcess.ProcessName));
```

Comment:
Why it is `Console.WriteLine` not `ConsoleOutput.Instance.WriteLine`?

Reply:
This is MSBuild task. We don't reference ConsoleOutput there.

Reply:
Yeah, please use `ConsoleOutput.Instance.WriteLine` if possible :) Easier for me to downgrade that way.